### DISCUSSION TITLE
How to change to the default theme

### DISCUSSION THREAD

**Discussion by pannet1** [LazyVim/LazyVim#740] [Q&A] [answered]
As always the work of Folke is awesome. Lazy is one of the best nvim, i have used ever.  if i press <LuC>, i get the theme preview and selected default. i also removed gruvebox theme which i copied from the example. 

How to set default theme, i am previewing, permanently.

**Comment by folke** [comment, marked as answer]
https://www.lazyvim.org/plugins/colorscheme

**Reply by pannet1** [reply]
thanks. it worked

![image](https://github.com/LazyVim/LazyVim/assets/12350276/53b59682-97e8-41d8-893a-ed64bd3b2a5f)
